src/components/expanded-state/chart/chart-data-labels/ChartDateLabel.js

```diff
@@ -16,18 +17,18 @@ const Label = styled(ChartXLabel)`
 `;
 
 const MONTHS = [
-  'Jan',
-  'Feb',
-  'Mar',
-  'Apr',
-  'May',
-  'Jun',
-  'Jul',
-  'Aug',
-  'Sep',
-  'Oct',
-  'Nov',
-  'Dec',
+  lang.t('expanded_state.chart.date.months.month_00'),
```

Comment:
any reason to index these starting from `00` instead of `01` (which more closely matches up with the actual month)?

Reply:
@jinchung I think it might make sense to use `00` here since `MONTHS` is indexed using `date.getMonth()` later in the file, and `getMonth()` indexes with January as 0. Therefore `Months[0]` would line up with the i18n tag `month_00`, both representing January. Let me know if I should change it back though, thanks!